packages/components/src/mobile/image-with-focalpoint/index.native.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { Image } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect, memo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+
+const ImageWithFocalPoint = ( {
+	containerSize,
+	contentHeight,
```

Comment:
If I remember correctly from previous experiments, you can make the image fill the container with absolute positioning without knowing the exact container dimensions in code. We would still have to listen to layout events and get the exact dimensions to calculate offset, but it might be better to keep that logic inside this component than passing dimensions from the outside.

Reply:
Definitely better to have the logic within the component, simpler to just pass the `url` and `focalPoint` and let the component handle the rest 🙌  https://github.com/WordPress/gutenberg/pull/19722/commits/e71ee485de0b4b67bbe5fb5f5d8c02f98ad9df7b